playwright/features/headerLinks.feature

```diff
@@ -18,32 +18,45 @@ Feature: Header Links
-            | "license"           |
-            | "about"             |  
+        Examples:
+            | DRB               |
+            | "home"            |
+            | "advanced search" |
+            | "search results"  |
+            | "item details"    |
+            | "edition details" |
+            | "collection"      |
+            | "read online"     |
+            | "license"         |
+            | "about"           |
 
     Scenario: As a user I navigate to the Digital Research Books home page and verify the account and search header sub-links and elements are displayed
         Given I go to the "home" page
         When I click the <first header link>
         Then the <second header link> should be displayed
 
         Examples:
-            | first header link         | second header link                       |
-            | "my account header link"  | "catalog header link"                    |
-            | "my account header link"  | "research catalog header link"           |
-            | "my account header link"  | "close my account header link"           |
-            | "search header link"      | "search header label"                    |
-            | "search header link"      | "search header text field"               |
-            | "search header link"      | "search books music movies radio button" |
-            | "search header link"      | "search research catalog radio button"   |
-            | "search header link"      | "search library website radio button"    |
-            | "search header link"      | "search header button"                   |
-            | "search header link"      | "close search header link"               |
+            | first header link        | second header link                       |
+            | "my account header link" | "catalog header link"                    |
+            | "my account header link" | "research catalog header link"           |
+            | "my account header link" | "close my account header link"           |
+            | "search header link"     | "search header label"                    |
+            | "search header link"     | "search header text field"               |
+            | "search header link"     | "search books music movies radio button" |
+            | "search header link"     | "search research catalog radio button"   |
+            | "search header link"     | "search library website radio button"    |
+            | "search header link"     | "search header button"                   |
+            | "search header link"     | "close search header link"               |
+
+    Scenario: As a user when I click on Locations I should be directed to NYPL location page
+        Given I go to the "home" page
+        Then I click the <first header link>
+        And the <landing page header> should be displayed
+
+        Examples:
+            | first header link           | landing page header              |
```

Comment:
I am not familiar with cucumber syntax. Does it always start with a row without double quotes?

Reply:
@Toxiapo yes. First row will always be without quotes to map the next rows with the main steps